docs/sql-reference/sql-functions/array-functions/cardinality.md

```diff
@@ -0,0 +1,37 @@
+# cardinality
+
+## Description
+
+Returns the number of elements in the array. The result type is INT. if the parameter is NULL, the result is also NULL.
```

Comment:
what if null element

Reply:
it is a element, so counting it in size.